<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>701</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btnUbah">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>570</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Ubah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSimpan">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>570</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>571</width>
     <height>321</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="iDAnggotaLabel">
      <property name="text">
       <string>ID Kasus</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="editID"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="namaAnggotaLabel">
      <property name="text">
       <string>ID Client</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="statusLabel">
      <property name="text">
       <string>Judul Kasus</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="editJudul"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="telpLabel">
      <property name="text">
       <string>Deskripsi / Uraian Kasus</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="emailLabel">
      <property name="text">
       <string>Tanggal Masuk</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="editTanggal"/>
    </item>
    <item row="3" column="1">
     <widget class="QTextEdit" name="editUraian"/>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboClient"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnHapus">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>570</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tabelKasus">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>380</y>
     <width>721</width>
     <height>151</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>ID Kasus</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ID Client</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Judul Kasus</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Uraian</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string> Tanggal Masuk</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="editCari">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>571</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
